components/SDL2/sdl2.p5m

```diff
@@ -0,0 +1,127 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/$(MACH64)/sdl2-config
+file path=usr/bin/sdl2-config
+file path=usr/lib/$(MACH64)/pkgconfig/sdl2.pc
+file path=usr/lib/pkgconfig/sdl2.pc
+file path=usr/lib/$(MACH64)/libSDL2-2.0.so.0.4.0
+link path=usr/lib/$(MACH64)/libSDL2-2.0.so.0 target=libSDL2-2.0.so.0.4.0
+link path=usr/lib/$(MACH64)/libSDL2-2.0.so target=libSDL2-2.0.so.0
+link path=usr/lib/$(MACH64)/libSDL2.so target=libSDL2-2.0.so
+file path=usr/lib/$(MACH64)/libSDL2.a
+file path=usr/lib/$(MACH64)/libSDL2_test.a
+file path=usr/lib/$(MACH64)/libSDL2main.a
+file path=usr/lib/libSDL2-2.0.so.0.4.0
+link path=usr/lib/libSDL2-2.0.so.0 target=libSDL2-2.0.so.0.4.0
+link path=usr/lib/libSDL2-2.0.so target=libSDL2-2.0.so.0
+link path=usr/lib/libSDL2.so target=libSDL2-2.0.so
+file path=usr/lib/libSDL2.a
+file path=usr/lib/libSDL2_test.a
+file path=usr/lib/libSDL2main.a
+dir  path=usr/lib/cmake
```

Comment:
It doesn't matter, file action for usr/lib/cmake/SDL2/sdl2-config.cmake is enough